druid/src/command.rs

```diff
@@ -172,18 +173,15 @@ pub mod sys {
     /// System command. A file picker dialog will be shown to the user, and an
     /// [`OPEN_FILE`] command will be sent if a file is chosen.
     ///
-    /// The argument should be a [`FileDialogOptions`] struct.
```

Comment:
I'm not sure that this should be removed. How do developers know that `FileDialogOptions` is the payload type? Should probably reword it from *argument* to *payload* though.

Reply:
The type is visible in the docs, auto completion (RustAnalyzer) and you will get an error if you pass anything else. I think that is enough.